sql-cli/sql_cli/project.py

```
@@ -124,7 +127,9 @@ def load_config(self, environment: str = DEFAULT_ENVIRONMENT) -> None:
         :param environment: string referencing the desired environment, uses "default" unless specified
         """
         if not self.is_valid_project():
-            logging.exception("This is not a valid SQL project. Please, use `flow init`")
+            raise Exception("This is not a valid SQL project. Please, use `flow init`")
```

We should probably raise a custom exception